<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseStationForm</class>
 <widget class="QDialog" name="BaseStationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="okCancelButtonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="possitionLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Possition</string>
   </property>
  </widget>
  <widget class="QLabel" name="heightLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height [m n.p.t.]</string>
   </property>
  </widget>
  <widget class="QLabel" name="xLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lon:</string>
   </property>
  </widget>
  <widget class="QLabel" name="yLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lat:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="heightBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="xSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>50.323500000000003</double>
   </property>
   <property name="maximum">
    <double>51.205100000000002</double>
   </property>
   <property name="singleStep">
    <double>0.000100000000000</double>
   </property>
   <property name="value">
    <double>51.177199999999999</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ySpinBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>15.974200000000000</double>
   </property>
   <property name="maximum">
    <double>17.412600000000001</double>
   </property>
   <property name="singleStep">
    <double>0.000100000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>BaseStationForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>BaseStationForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
